OptiFineDoc/assets/minecraft/optifine/lang/pt_br.lang

```diff
@@ -57,7 +57,7 @@ of.options.capeOF.title=Capa do OptiFine
 of.options.capeOF.openEditor=Abrir editor de capa
 of.options.capeOF.reloadCape=Recarregar capa
 
-of.message.capeOF.openEditor=O editor de capas do OptiFine será aberto num navegador.
+of.message.capeOF.openEditor=O editor de capas do OptiFine será aberto em um navegador.
```

Comment:
Unless we have lots of free space to fit those texts (we haven't), the use of informal style is a resource to avoid overflows.